### DISCUSSION TITLE
app.config undefined in boot file

### DISCUSSION THREAD

**Discussion by MichelPWeber** [quasarframework/quasar#9367] [General - Components / Directives / etc] [answered]
I created a new boot file.
app.use isn't available so using app.config.globalProperties is what I'm trying to use.  But I get an error in the browser:
### [Quasar] boot error: TypeError: Cannot read property 'globalProperties' of undefined

Here is the boot file contents:

```
// import something here
import GoogleAuth from '../config/google_oAuth.js'

// "async" is optional;
// more info on params: https://quasar.dev/quasar-cli/boot-files
export default ({ app } /* { app, router, Vue ... } */) => {
  // something to do
  app.config.globalProperties.$gAuth = GoogleAuth
  // app.use(GoogleAuth)
}
```

Not sure how to make use of GoogleAuth

**Comment by TobyMosque** [comment]
in order to use Vue3, u'll need quasar v2.
just type `quasar info` in your console to check out what version u're using.

**Comment by MichelPWeber** [comment, marked as answer]
Resolved - used Vue.use instead.
